plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/messages.properties

```diff
@@ -161,47 +161,47 @@ NavigatorGroupName_SpringMediatorOutputConnector_3062_outgoinglinks=outgoing lin
 NavigatorGroupName_DBLookupMediatorOutputConnector_3077_outgoinglinks=outgoing links
 NavigatorGroupName_SmooksMediatorOutputConnector_3083_outgoinglinks=outgoing links
 SendMediator19CreationTool_title=Send
-SendMediator19CreationTool_desc=Create new Send Mediator
+SendMediator19CreationTool_desc=Sends a message
 NavigatorGroupName_SendMediatorInputConnector_3085_incominglinks=incoming links
 NavigatorGroupName_SendMediatorOutputConnector_3086_outgoinglinks=outgoing links
 FailoverEndPoint3CreationTool_title=FailoverEndPoint
-FailoverEndPoint3CreationTool_desc=Create new Failover EndPoint
+FailoverEndPoint3CreationTool_desc=Defines the endpoints that the service will try to connect to in case of a failure. This will take place in a round robin manner
 NavigatorGroupName_FailoverEndPointInputConnector_3088_incominglinks=incoming links
 NavigatorGroupName_FailoverEndPointOutputConnector_3090_outgoinglinks=outgoing links
 WSDLEndPoint4CreationTool_title=WSDLEndPoint
-WSDLEndPoint4CreationTool_desc=Create new WSDL EndPoint
+WSDLEndPoint4CreationTool_desc=Defines the WSDL, Service and Port
 NavigatorGroupName_WSDLEndPointOutputConnector_3093_outgoinglinks=outgoing links
 NavigatorGroupName_WSDLEndPointInputConnector_3092_incominglinks=incoming links
 LoadBalanceEndPoint5CreationTool_title=Load-BalanceEndPoint
-LoadBalanceEndPoint5CreationTool_desc=Create new Load-Balance EndPoint
+LoadBalanceEndPoint5CreationTool_desc=Defines groups of endpoints for replicated services.The incoming requests will be directed to these endpoints in a round robin manner. These endpoints automatically handle the fail-over cases as well
 NavigatorGroupName_LoadBalanceEndPointOutputConnector_3096_outgoinglinks=outgoing links
 NavigatorGroupName_LoadBalanceEndPointInputConnector_3095_incominglinks=incoming links
 NavigatorGroupName_LoadBalanceEndPointWestOutputConnector_3098_outgoinglinks=outgoing links
 NavigatorGroupName_FailoverEndPointWestOutputConnector_3097_outgoinglinks=outgoing links
 HeaderMediator20CreationTool_title=Header
-HeaderMediator20CreationTool_desc=Create new Header Mediator
+HeaderMediator20CreationTool_desc=Sets or removes SOAP headers
```

Comment:
Header mediator supports both SOAP and transport headers

Reply:
Fixed.